src/components/reusable/textArea/textArea.scss

```diff
@@ -24,11 +24,14 @@ textarea {
   right: 16px;
 }
 
-.count {
-  @include typography.type-ui-03;
-  position: absolute;
-  right: 22px;
-  bottom: 6px;
-  padding: 2px;
-  background: var(--kd-color-background-inverse);
+.caption-error-count {
+  display: flex;
```

Comment:
Add a `gap: 16px;` or similar, so the caption doesn't run right into the count if long enough.

Reply:
Done